azext_iot/tests/digitaltwins/__init__.py

```diff
@@ -269,7 +285,8 @@ def ensure_adx_resource(self):
                 retries += 1
 
             if retries == MAX_ADX_RETRIES:
-                logger.error("Waited 2.5 minutes for database creation and database has not been created yet.")
+                database_msg = f"Waited 2.5 minutes for database creation and database {ADX_DATABASE} has not been created yet."
```

Comment:
The time waited can be a variable/dynamically calcualted so it doesn't have to be manually updated if the retries/sleep intervals change.

Section higher up with the same suggestion.